### PR TITLE
Git hooks and github actions

### PR DESCRIPTION
**Pull request by brentmiller1973**
I addressed the npm run test -- --changedSince=${{ github.head_ref }} issue (thanks @matthewroach ).  Not sure why the test is still failing however.  The git hooks are only hard coded to the base branch, spent too much time trying to figure out how to lint and test against base dynamically....I'm calling mercy.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,8 +6,11 @@
   "scripts": {
     "test": "jest --coverage",
     "test:watch": "npm run test -- --watch -o --coverage",
+    "test:changed-feature-branch": "jest --changedSince=github.head_ref --coverage --passWithNoTests",
```
Comment (matthewroach) [inline package.json]: This is failing in the action as the value for `--changedSince` needs to be passed via the action like

```npm run test -- --changedSince=${{ github.head_ref }}```

**Review by matthewroach — CHANGES_REQUESTED** [review]
The action is failing due to an issue with the `--changedSince` value which I added a comment too

**Comment by matthewroach** [comment]
@visionbegin - There was some issue with the Chromatic action I rebased from the base branch `arc-themes-release-version-1.19` - you'll need to pull from origin to get the latest updates

Regarding the changed since parameter - I've updated that to work - it needed to be `base_ref` not `head_ref` I made a mistake and got confused between the two keys and what values they provided.

Take a look over the PR and see if it all makes sense now?

**Review by matthewroach — APPROVED** [review]

**Review by matthewroach — APPROVED** [review]

**Comment by brentmiller1973** [comment]
> @visionbegin - There was some issue with the Chromatic action I rebased from the base branch `arc-themes-release-version-1.19` - you'll need to pull from origin to get the latest updates
> 
> Regarding the changed since parameter - I've updated that to work - it needed to be `base_ref` not `head_ref` I made a mistake and got confused between the two keys and what values they provided.
> 
> Take a look over the PR and see if it all makes sense now?

@matthewroach Thank you so much for the guidance!

**Pull request merged**